src/logging.c

```diff
@@ -31,6 +38,19 @@ void rcutils_logging_initialize()
   if (!g_rcutils_logging_initialized) {
     g_rcutils_logging_output_handler = &rcutils_logging_console_output_handler;
     g_rcutils_logging_severity_threshold = RCUTILS_LOG_SEVERITY_INFO;
+
+    int max_output_format_len = sizeof(g_rcutils_logging_output_format_string);
+    memset(g_rcutils_logging_output_format_string, '\0', max_output_format_len);
```

Comment:
It is not necessary to overwrite all characters with `\0`. The first one would be sufficient to make it a null terminated string.

Same below.